Resources/Prototypes/_Goobstation/Entities/Clothing/OuterClothing/hardsuits.yml

```diff
@@ -94,3 +94,53 @@
         # SIGNIFICANTLY Quieter than a normal hardsuit, but not perfectly stealthy. If you move while people are close enough, they can still hear it.
   - type: StaminaDamageResistance
     coefficient: 0.85 # 15%. Substantially lower than a normal hardsuit.
+
+# cybersun dreadnought
+
+- type: entity
+  parent: ClothingOuterHardsuitBase
+  id: ClothingOuterHardsuitCybersunDreadnought
+  name: cybersun dreadnought suit
+  description: You are wearing AND carrying Cybersun's glory on your shoulders. Do not disappoint.
```

Comment:
```suggestion
  name: CSA-105UA "Xíngtiān" tacsuit # https://en.wikipedia.org/wiki/Xingtian in case you were wondering where the name comes from.
  description: A rare and exotic horror from the minds of Sol's finest engineers. This suit is worn as a punishment for crimes against Mother Sol.
```